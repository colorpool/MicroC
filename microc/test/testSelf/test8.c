int main(int n){
    // print 'd' n;
    // print 'd' n++;
    // print 'd' n;
    // print 'd' ++n;
    // print 'd' n;
    // print 'd' n--;
    // print 'd' n;
    print 'd' --n;
    // print 'd' n;
}