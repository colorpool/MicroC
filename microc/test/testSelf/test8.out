24 19 1 5 25 13 0 1 2 15 -1 13 13 11 0 1 2 12 22 21 1